packages/ui-editable/src/InPlaceEdit/index.tsx

```diff
@@ -166,12 +173,16 @@ class InPlaceEdit extends Component<InPlaceEditProps> {
       this.props.editButtonPlacement === 'start'
         ? '0 xx-small 0 0'
         : '0 0 0 xx-small'
+
+    const { ['ref']: omitRef, ...omitRefFromContainerProps } =
+      getContainerProps()
     return (
       <Flex
         display={this.props.inline ? 'inline-flex' : 'flex'}
         direction={flexDir}
         justifyItems={justifyItems}
-        {...getContainerProps()}
+        {...omitRefFromContainerProps}
+        {...(omitRef as unknown as (el: Flex) => void)}
```

Comment:
I think this TS error shows that the `ref` is indeed not typed correctly in the `getContainerProps`, and it should be fixed instead. I would revert this change and type the `ref` like this:

```tsx
type containerProps = (props?: Record<string, any>) => {
  ref: React.RefCallback<any>
  ...
}
```